src/common/marlin_server.hpp

```diff
@@ -87,8 +87,8 @@ class FSM_notifier {
 template <int VAR_ID, class T>
 class Notifier : public FSM_notifier {
 public:
-    Notifier(ClientFSM type, uint8_t phase, T min, T max, uint8_t progress_min, uint8_t progress_max)
```

Comment:
do not change signature.
U can use partial template specialization for T as float